handlers/abtest/search.go

```diff
@@ -0,0 +1,83 @@
+package abtest
+
+import (
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/ONSdigital/dp-cookies/cookies"
+	"github.com/ONSdigital/log.go/log"
+)
+
+//go:generate moq -out searchtest/handler.go -pkg test . Handler
+type Handler http.Handler
+
+func SearchHandler(newSearch, oldSearch http.Handler, percentage int, domain string) http.Handler {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I assume this line can be removed

Reply:
I think it's needed for truly random numbers?

>Top-level functions, such as Float64 and Int, use a default shared Source that produces a deterministic sequence of values each time a program is run. Use the Seed function to initialize the default Source if different behavior is required for each run.

Reply:
Ah okay, I wasn't sure how this was being used but rand is the receiver for Seed method.

Could I suggest moving this line to where we act on the receiver on line 64 of this file?